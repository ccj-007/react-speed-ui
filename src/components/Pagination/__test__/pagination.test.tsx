import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Pagination } from '../../index';

describe('test Pagination component', () => {
  it('should render', () => {
    const { asFragment } = render(<Pagination />);
    expect(asFragment()).toMatchSnapshot();
  });
});
