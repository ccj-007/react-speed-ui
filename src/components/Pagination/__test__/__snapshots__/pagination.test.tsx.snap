// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Pagination component should render 1`] = `
<DocumentFragment>
  <div
    class="speed-pagination"
  >
    <div
      class="speed-pagination-page"
    >
      <svg
        aria-hidden="true"
        class="svg-inline--fa fa-angle-left fa-1x speed-icon"
        color="#000"
        data-icon="angle-left"
        data-prefix="fas"
        focusable="false"
        role="img"
        viewBox="0 0 320 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M41.4 233.4c-12.5 12.5-12.5 32.8 0 45.3l160 160c12.5 12.5 32.8 12.5 45.3 0s12.5-32.8 0-45.3L109.3 256 246.6 118.6c12.5-12.5 12.5-32.8 0-45.3s-32.8-12.5-45.3 0l-160 160z"
          fill="currentColor"
        />
      </svg>
    </div>
    <button
      class="speed-pagination-page"
    >
      1
    </button>
    <button
      class="speed-pagination-page speed-pagination-page-active"
    >
      2
    </button>
    <button
      class="speed-pagination-page"
    >
      3
    </button>
    <button
      class="speed-pagination-page"
    >
      4
    </button>
    <button
      class="speed-pagination-page"
    >
      5
    </button>
    <div
      class="speed-pagination-page"
    >
      <svg
        aria-hidden="true"
        class="svg-inline--fa fa-angle-right fa-1x speed-icon"
        color="#000"
        data-icon="angle-right"
        data-prefix="fas"
        focusable="false"
        role="img"
        viewBox="0 0 320 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M278.6 233.4c12.5 12.5 12.5 32.8 0 45.3l-160 160c-12.5 12.5-32.8 12.5-45.3 0s-12.5-32.8 0-45.3L210.7 256 73.4 118.6c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0l160 160z"
          fill="currentColor"
        />
      </svg>
    </div>
  </div>
</DocumentFragment>
`;
